### DISCUSSION TITLE
Image size in populate results?

### DISCUSSION THREAD

**Discussion by dhaillant** [yaqwsx/PcbDraw#110] [Q&A]
Hello,
Is there a way to set the size of the images in populate results?
Thank you!

**Comment by yaqwsx** [comment]
Do you mean in page or the actual sources? If you want to change it on the page, just supply a custom HTML template. The source size is controlled via the DPI option in header (the options that are directly passed to Pcbdraw)

**Comment by dhaillant** [comment]
Either in the web page or in the actual images, whichever is the easiest.
The custom HTML template is probably the way to go I guess?

But I was unable to set the DPI option in the first place on command line, the option doesn't seem to be accepted by populate:

pcbdraw populate ./populate.txt img/ --dpi 100
Error: No such option: --dpi

By 'header', do you mean in the markdown file?

Could you provide an example please? Thank you very much!
